### PR TITLE
5 task 2 profile

### PR DESCRIPTION
**Pull request by coccooma**
implemented profile feature

### CODE REVIEW

File: spec/models/representative_spec.rb [hunk 1]
```diff
@@ -24,5 +31,44 @@
       described_class.civic_api_to_representative_params(rep_info)
       expect(described_class.count).to eq(1)
     end
+
+    it 'creates or updates Representative records with the correct street and city parameters' do
```
[the remark this answers is not in the document]
Comment (coccooma) [reply spec/models/representative_spec.rb]: fixed

File: spec/models/representative_spec.rb [hunk 2]
```diff
@@ -24,5 +31,44 @@
...
+      @representatives.each do |representative|
+        expect(representative).to be_persisted
+        expect(representative.street).to eq('123 Main St')
+        expect(representative.city).to eq('Los Angeles')
+      end
+    end
+
+    it 'creates or updates Representative records with the correct name and ocdid parameters' do
```
[the remark this answers is not in the document]
Comment (coccooma) [reply spec/models/representative_spec.rb]: fixed

File: spec/models/representative_spec.rb [hunk 3]
```diff
@@ -24,5 +31,44 @@
...
+      @representatives.each do |representative|
+        expect(representative).to be_persisted
+        expect(representative.name).to eq('Karen Bass')
+        expect(representative.ocdid).to eq('ocd-division/country:us/state:ca/city:los_angeles')
+      end
+    end
+
+    it 'creates or updates Representative records with the correct zip and party parameters' do
```
[the remark this answers is not in the document]
Comment (coccooma) [reply spec/models/representative_spec.rb]: fixed

File: spec/models/representative_spec.rb [hunk 4]
```diff
@@ -24,5 +31,44 @@
...
+      @representatives.each do |representative|
+        expect(representative).to be_persisted
+        expect(representative.zip).to eq('90001')
+        expect(representative.political_party).to eq('Democrat')
+      end
+    end
+
+    it 'creates or updates Representative records with the correct state and title parameters' do
```
[the remark this answers is not in the document]
Comment (coccooma) [reply spec/models/representative_spec.rb]: fixed

File: spec/models/representative_spec.rb [hunk 5]
```diff
@@ -24,5 +31,46 @@
       described_class.civic_api_to_representative_params(rep_info)
       expect(described_class.count).to eq(1)
     end
+
+    context 'when creates or updates Representative records with the correct' do
+      it 'street and city parameters' do
```
[the remark this answers is not in the document]
Comment (coccooma) [reply spec/models/representative_spec.rb]: fix

File: spec/models/representative_spec.rb [hunk 6]
```diff
@@ -24,5 +31,46 @@
...
+        @representatives.each do |representative|
+          expect(representative).to be_persisted
+          expect(representative.street).to eq('123 Main St')
+          expect(representative.city).to eq('Los Angeles')
+        end
+      end
+
+      it 'name and ocdid parameters' do
```
[the remark this answers is not in the document]
Comment (coccooma) [reply spec/models/representative_spec.rb]: fix

File: spec/models/representative_spec.rb [hunk 7]
```diff
@@ -24,5 +31,46 @@
...
+        @representatives.each do |representative|
+          expect(representative).to be_persisted
+          expect(representative.name).to eq('Karen Bass')
+          expect(representative.ocdid).to eq('ocd-division/country:us/state:ca/city:los_angeles')
+        end
+      end
+
+      it 'zip and party parameters' do
```
[the remark this answers is not in the document]
Comment (coccooma) [reply spec/models/representative_spec.rb]: fix

File: spec/models/representative_spec.rb [hunk 8]
```diff
@@ -24,5 +31,46 @@
...
+        @representatives.each do |representative|
+          expect(representative).to be_persisted
+          expect(representative.zip).to eq('90001')
+          expect(representative.political_party).to eq('Democrat')
+        end
+      end
+
+      it 'state and title parameters' do
```
[the remark this answers is not in the document]
Comment (coccooma) [reply spec/models/representative_spec.rb]: fix

**Review by tracyye029 — COMMENTED** [review]
looks good to me for now

**Comment by bsmider** [comment]
lgtm

**Pull request merged**